HttpResponse.java

```diff
@@ -0,0 +1,16 @@
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+interface HttpResponse {
+    void setMessageBody(InputStream body) throws IOException;
+    void setVersion(String version);
+    void setStatusCode(int StatusCode);
+    void setStatusMessage(String StatusMessage);
+    void appendHeader(String llave, String valor);
+
+    OutputStream getoutputStream();
```

Comment:
Para mantener la consistencia deberia de llamarse getOutptutStream();
```suggestion
    OutputStream getOutputStream();
```